lib/okta/get-app-user-by-id.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const requestExecutor = require('./request-executor');
+
+function getAppUserById(org, appId, apiToken, uid, callback) {
+  const req = {
+    url: org + '/api/v1/apps/' + appId + '/users/' + uid,
```

Comment:
Heh, as an aside, when moving to es6 it's really handy to use template literals for things like this, i.e.
```javascript
url: `${org}/api/v1/apps/${appId}/users/${uid}`
```